sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/autorest.md

```diff
@@ -8,7 +8,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 ``` yaml
 tag: package-artifacts-composite-v1
```

Comment:
shouldn't the tag be composite-v2 ?

Reply:
Nice catch @g2vinay!  I will update it, thank you so much!